### ISSUE TITLE
helm completion does not support commands aliases

### ISSUE THREAD

**Issue by thomas-riccardi** [helm/helm#3171]
```bash
# OK
$ helm list --<TAB>
--all                --date               --debug              --deleted            --deleting           --deployed           --failed             --home=              --host=              --kube-context=      --max=
--namespace=         --offset=            --pending            --reverse            --short              --tiller-namespace=  --tls                --tls-ca-cert=       --tls-cert=          --tls-key=           --tls-verify

# KO
$ helm ls --<TAB>
--debug              --home=              --host=              --kube-context=      --tiller-namespace=
```

**Comment by rajatjindal** [comment]
FYI, I've submitted a PR in spf13/cobra which 'should' address this issue.

https://github.com/spf13/cobra/pull/588

**Comment by rajatjindal** [comment]
# FIXED ALIAS
$ helm ls --<TAB>
--all                --deleted            --failed             --kube-context=      --namespace=         --reverse            --tls                --tls-key=
--date               --deleting           --home=              --kubeconfig=        --offset=            --short              --tls-ca-cert=       --tls-verify
--debug              --deployed           --host=              --max=               --pending            --tiller-namespace=  --tls-cert=

# ORIGINAL
$ helm list --<TAB>
--all                --deleted            --failed             --kube-context=      --namespace=         --reverse            --tls                --tls-key=
--date               --deleting           --home=              --kubeconfig=        --offset=            --short              --tls-ca-cert=       --tls-verify
--debug              --deployed           --host=              --max=               --pending            --tiller-namespace=  --tls-cert=

This is fixed by https://github.com/spf13/cobra/pull/588

**Comment by thomas-riccardi** [comment]
fix in cobra was merged (https://github.com/spf13/cobra/pull/588) but cobra dependency in helm was *not* updated.

**Comment by thomas-riccardi** [comment]
I still reproduce the issue with helm 2.9.0.

**Comment by bacongobbler** [comment]
We still need to bump cobra to bring in this fix. Re-opening!

**Comment by marckhouzam** [comment]
This can be closed.
Cobra has been updated a couple of time since this was a problem.
I believe this was fixed when #4913 was merged.
It is part of Helm 2.12 if I'm not mistaken.
